en/computer_vision/collision_prevention.md

```diff
@@ -204,3 +204,4 @@ See [PX4/PX4-Avoidance](https://github.com/PX4/PX4-Avoidance) for setup instruct
 
 <!-- PR companion collision prevention (initial): https://github.com/PX4/PX4-Autopilot/pull/10785 -->
 <!-- PR for FC sensor collision prevention: https://github.com/PX4/PX4-Autopilot/pull/12179 -->
+<!-- using rangefinder? -->
```

Comment:
> 5. We should certainly update https://docs.px4.io/main/en/computer_vision/collision_prevention.html#px4-distance-sensor right?
 >> 5. see (2.)

So I've tagged this section at the end of the Collision parameters page as an explanation of what I mean.

My assumption is that you added this for testing collision prevention. So it might be good to remove this section pointing to the PX4-Avoidance repo, which is broken, and replace it with a section that explains how you can test collision prevention using this new lidar.

Do you know. IF not, I consider this non-blocking, but would be great if you can tell me who might know? (or I can try it myself).

Reply:
i did add a small section, might be expanded once i'm done with collision prevention